nss-defective-foxes/nutshell
# Cr initial chat structure

## Link to Ticket
Closes # [https://github.com/nss-defective-foxes/nutshell/projects/1#card-9815899]()

## Description of Proposed Changes

- Chat must display messages
- Chat must load input and submit button
- Button must repopulate messages with new message on bottom
- Must find appropriate user and populate name before the message
- Text inserted in input must match text of messages posted in chatboard

## Steps to Test

- Initial messages of dummy data should populate once hs -o first loads
- Currently with no log in you MUST set a user id in session storage to an EXISTING user id
- Once that is completed (can be done through the console in devtools) attempt to type a message
- Press submit button when done with message
- Message with userId preppended should populate at the bottom of the chat
- The input box should appear empty
- Dummy data should be updated if viewed in the messages local host

Outline the steps to test

```sh
git fetch --all
git checkout CRInitialChatStructure
```



## Impacted Areas in Application

*Chat box and webpage layout

## Mentions @username

@AnEvilHerbivore 
@wcdoster 
@sirwin4
